en/protocol/ftp.md

```diff
@@ -0,0 +1,64 @@
+# File Transfer Protocol (FTP)
+
+> **Note** THIS IS WORK IN PROGRESS: The FTP protocol is pretty simple. Right now everything is exchanged inside [ENCAPSULATED_DATA](../messages/common.md#MAV_PROTOCOL_CAPABILITY_FTP) packets, and there are a few other things that could be tightened up, but the protocol should be brought up first before optimising bandwidth. 
+
+The actual implementation is in this file:
+* [src/modules/mavlink/mavlink_ftp.cpp](https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_ftp.cpp)
+* [src/modules/mavlink/mavlink_ftp.h](https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_ftp.h)
+
+The QGC side of the implementation is here:
+* [src/uas/FileManager.cc](https://github.com/mavlink/qgroundcontrol/blob/master/src/uas/FileManager.cc)
+
+And with the main protocol being implemented here (which has a few function calls for test purposes):
```

Comment:
This link broken. Is this architecture still correct, and if not is this needed?